Jenkinsfile

```diff
@@ -548,6 +578,20 @@ pipeline {
                         buildHipClangJobAndReboot( setup_flags: Tensile_setup + extra_log_env + Bf16_flags, build_env: Tensile_build_env + extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx908:xnack-", miotensile_version: Tensile_version, target_id: "ON")
                     }
                 }
+                // stage('Fp32 Hip Tensile-Latest All gfx90a') {
```

Comment:
What about these stages?

Reply:
They are tracked by this issue:
https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1094